modules/ROOT/nav.adoc

```diff
@@ -74,6 +74,7 @@
   ** xref:track-loggedout-sso.adoc[Track logged-out SSO cookies]
   ** xref:configuring-spnego-authentication.adoc[Configuring SPNEGO authentication]
   ** xref:enable-openid-connect-client.adoc[Enable an OpenID Connect client]
+* xref:json-web-token.adoc[JSON Web Tokens (JWT)]
```

Comment:
windering if this should be nested in the "Single Sign-On" section?

Reply:
I didn't want it to be nested under Single Sign-On since I don't really consider JWTs to be an SSO technology. I'd be fine nesting it if you had a stronger opinion on it, though.

Reply:
interesting- ok we can keep it as you have it then